src/lib/db/feature-toggle-store.ts

```diff
@@ -262,7 +262,26 @@ export default class FeatureToggleStore implements IFeatureToggleStore {
             .where({ name: data.name })
             .update(this.dtoToRow(project, data))
             .returning(FEATURE_COLUMNS);
-        return this.rowToFeature(row[0]);
+
+        const feature = this.rowToFeature(row[0]);
+        // if a feature toggle's type or createdAt has changed, update its potentially stale status
+        if (!feature.stale && (data.type || data.createdAt)) {
+            await this.db(TABLE)
+                .where({ name: data.name })
+                .update(
+                    'potentially_stale',
+                    this.db.raw(
+                        `COALESCE (? > (features.created_at + ((
```

Comment:
Why coalesce?

Reply:
Just removing coalesce should work.

Reply:
Huh, thought I tried that, but .. fixed in 5e5a3d1c9a253740682eef4bad537e4bd0ab2949